### PR TITLE
Clarified and expanded E2C help text

### PR DESCRIPTION
**Pull request by TonyB9000**
I believe this clarifies and improves the help text.  There is still future work involving the parsing logic.  Specifically, "--info" will fail if "--map" is not specified (for mpas* realm), but map is not needed.  You can supply "--map stub" and it works fine.

### CODE REVIEW

**Review by chengzhuzhang — APPROVED** [review]
Looks good! Separately, we should also add an example for MPAS O variable in the documentation. Thanks for the PR!

**Comment by TonyB9000** [comment]
Is "the documentation" accessible via the repo, or is this a separate Confluence page?  I can try to create an example in the same style as the other examples, based perhaps upon the mpaso example your provided for Strand.  If the current docs were specially crafted, I may need to learn something.

**Comment by chengzhuzhang** [comment]
@TonyB9000 I think you can add the example through editing this file: https://github.com/E3SM-Project/e3sm_to_cmip/blob/master/docs/source/examples.rst but @tomvothecoder can correct me if the docs needs to be built manually.

**Comment by tomvothecoder** [comment]
> @TonyB9000 I think you can add the example through editing this file: https://github.com/E3SM-Project/e3sm_to_cmip/blob/master/docs/source/examples.rst but @tomvothecoder can correct me if the docs needs to be built manually.

That's correct. The `.rst` files related to the docs can be edited directly.  GitHub Actions will automatically build and publish the doc updates after the PR is merged.

**Comment by TonyB9000** [comment]
Awesomeness!  Thanks.

**Comment by TonyB9000** [comment]
@tomvothecoder  How do I "checkout a new branch" to propose changes to this "master"?  Can I git-clone (thing), create the branch etc - as usual?  I don't know how to do that in the git web page.

**Comment by chengzhuzhang** [comment]
Hey Tony, I'm pasting your comment in email for explaining commit [c368ca6](https://github.com/E3SM-Project/e3sm_to_cmip/pull/136/commits/c368ca6834255178bd4a954dc97dd24930be19da)

ISSUE:  The following command used to work, now does not.
 
(e2c_dev) -bash-4.2$ e3sm_to_cmip --info -i /p/user_pub/work/E3SM/1_0/hist-GHG/1deg_atm_60-30km_ocean/atmos/native/model-output/day/ens1/v1 --freq day -v pr_highfreq -t /p/user_pub/e3sm/staging/resource/cmor/cmip6-cmor-tables/Tables/ --info-out ztest_info_out
Traceback (most recent call last):
  File "/home/bartoletti1/anaconda3/envs/e2c_dev/bin/e3sm_to_cmip", line 8, in <module>
    sys.exit(main())
  File "/home/bartoletti1/anaconda3/envs/e2c_dev/lib/python3.8/site-packages/e3sm_to_cmip/__main__.py", line 93, in main
    handlers = _load_handlers(
TypeError: _load_handlers() got an unexpected keyword argument 'simple'
 
The “load_handlers()” is defined in “util.py”.  I removed “simple=simple” from the call to “load_handlers()” in main because “simple” does not appear in the load_handlers code.
 
SUCCESS:  I can apply either “pr_highfreq” or just “pr” (with freq = day), and either way I get the high-freq handler selected!
 
(e2c_dev) -bash-4.2$ e3sm_to_cmip --info -i /p/user_pub/work/E3SM/1_0/hist-GHG/1deg_atm_60-30km_ocean/atmos/native/model-output/day/ens1/v1 --freq day -v pr_highfreq -t /p/user_pub/e3sm/staging/resource/cmor/cmip6-cmor-tables/Tables/ --info-out ztest_info_out
[*] Loaded `pr_highfreq` handler for variable `pr`.
(e2c_dev) -bash-4.2$ e3sm_to_cmip --info -i /p/user_pub/work/E3SM/1_0/hist-GHG/1deg_atm_60-30km_ocean/atmos/native/model-output/day/ens1/v1 --freq day -v pr -t /p/user_pub/e3sm/staging/resource/cmor/cmip6-cmor-tables/Tables/ --info-out ztest_info_out2
[*] Loaded `pr_highfreq` handler for variable `pr`.
 
I have pushed the changes on e2c branch “update_e2c_helptext”

File: e3sm_to_cmip/__main__.py [hunk 1]
```diff
@@ -95,8 +95,7 @@ def main():
         tables_path=tables_path,
         var_list=var_list,
         freq=freq,
-        realm=realm,
-        simple=simple)
+        realm=realm)
```
Comment (tomvothecoder) [inline e3sm_to_cmip/__main__.py:99]: Awesome, thanks for fixing my oversight!

Comment (tomvothecoder) [inline e3sm_to_cmip/__main__.py:99]: Awesome, thanks for fixing my oversight!

File: e3sm_to_cmip/util.py [hunk 2]
```diff
@@ -140,92 +140,93 @@ def parse_arguments():
...
     parser.add_argument(
-        "--debug", help="Set output level to debug", action="store_true"
+        "--debug", help="Set output level to debug.", action="store_true"
     )
     parser.add_argument(
         "--realm",
         metavar="<realm>",
         default="atm",
-        help="The realm to process, atm, lnd, mpaso or mpassi. Default is atm",
+        help="The realm to process.  Must be atm, lnd, mpaso or mpassi. Default is atm.",
```
Comment (tomvothecoder) [inline e3sm_to_cmip/util.py:215]:
```suggestion
        help="The realm to process. Must be atm, lnd, mpaso or mpassi. Default is atm.",
```

File: e3sm_to_cmip/util.py [hunk 3]
```diff
@@ -140,92 +140,93 @@ def parse_arguments():
...
     parser.add_argument(
         "--timeout",
-        help="Exit with code -1 if execution time exceeds given time in seconds",
+        help="Exit with code -1 if execution time exceeds given time in seconds.",
     )
     parser.add_argument(
         "--precheck",
         type=str,
-        help="Check for each variable if its already in the output CMIP6 directory, only run variables that dont have CMIP6 output",
+        help="Check for each variable if its already in the output CMIP6 directory, only run variables that don't have pre-existing CMIP6 output.",
```
Comment (tomvothecoder) [inline e3sm_to_cmip/util.py:229]:
```suggestion
        help="Check for each variable if it's already in the output CMIP6 directory, only run variables that don't have pre-existing CMIP6 output.",
```

File: e3sm_to_cmip/util.py [hunk 4]
```diff
@@ -239,11 +240,11 @@ def parse_arguments():
     parser.add_argument(
         "--info-out",
         type=str,
-        help="If passed with the --info flag, will cause the variable info to be written out to the specified file path as yaml",
+        help="If passed with the --info flag, will cause the variable info to be written out to the specified file path as yaml.",
     )
     parser.add_argument(
         "--version",
-        help="print the version number and exit",
+        help="print the version number and exit.",
```
Comment (tomvothecoder) [inline e3sm_to_cmip/util.py:247]:
```suggestion
        help="Print the version number and exit.",
```

**Review by tomvothecoder — CHANGES_REQUESTED** [review]
Thanks Tony. I had some minor code suggestions to make. You can go to the Files Changed tab and and batch the code suggestions to push them in a single commit.

File: e3sm_to_cmip/__main__.py (hunk 1 above)
Comment (TonyB9000) [reply e3sm_to_cmip/__main__.py:99]: no prob - almost gives the impression I know what I'm doing...

**Comment by TonyB9000** [comment]
"You can go to the Files Changed tab and and batch the code suggestions to push them in a single commit."

I think I did that right ... Now what?

**Review by tomvothecoder — APPROVED** [review]

**Comment by TonyB9000** [comment]
Yeah - I'll need to look for that next time.  I never saw it.  Thanks.

**Comment by TonyB9000** [comment]
I'm deleting the "branch can be safely deleted"...

**Pull request merged**